prescriptions/2p_/2ping/f35_rpm.yaml

```diff
@@ -0,0 +1,16 @@
+units:
+wraps:
+    - name: 2pingFedora35RPMWrap
+    type: wrap
+    should_include:
+        adviser_pipeline: true
```

Comment:
Could you please include the operating system here? That way, we show this information only for matching environments:

```yaml
      runtime_environments:
        operating_systems:
        - name: fedora
          version: "35"
```